src/encoded/schemas/changelogs/library.md

```diff
@@ -2,6 +2,8 @@
 
 ### Minor changes since schema version 16
 * Calculated properties *replicates* and *antibodies* are removed.
+* Added *Parse Single Cell Whole Transcriptome Kit* to the enum list for *construction_method*
+* *Parse Single Cell Whole Transcriptome Kit* was added to the enum list for *construction_method*
```

Comment:
This changelog is duplicated?

Reply:
Thanks for catching this- I think I missed it when going through a second commit checking merge conflicts in my rebase